contrib/debian/bitcoind.manpages

```diff
@@ -1,3 +1,4 @@
-debian/manpages/bitcoind.1
-debian/manpages/bitcoin.conf.5
-debian/manpages/bitcoin-cli.1
+doc/man/bitcoind.1
+doc/man/bitcoin-cli.1
+doc/man/bitcoin-qt.1
```

Comment:
This should be in a contrib/debian/bitcoin-qt.manpages (and similarly for bitcoin-tx)